backend/app/api/v1/endpoints/presignedurl.py

```
@@ -0,0 +1,34 @@
+from app.middlewares.auth_middleware import auth_required
+from app.schemas.files import PresignedDownloadRequest, PresignedUploadRequest
+from app.services.external.aws.s3 import S3Client
+from app.services.internal.file_upload import FileService
+from app.utils.response_handler import ResponseHandler
+
+router = APIRouter()
+
+
+@router.post("")
+@auth_required()
+async def generate_presigned_url(request: Request, body: PresignedUploadRequest):
```

Ruff (0.14.13)

Unused function argument: `request`

(ARG001)